### PR TITLE
Added orbit module for tomcat version upgrade from 7.0.70wso2v1 to 7.0.73wso2v1

### PR DESCRIPTION
**Pull request by NipunaMarcus**
There are new security fixes which are available for tomcat 7.0.73. This version upgrade is created to address those. For more details on what change in tomcat 7.0.73 visit following links.

[1].https://tomcat.apache.org/security-7.html#Fixed_in_Apache_Tomcat_7.0.73
[2].http://tomcat.apache.org/tomcat-7.0-doc/changelog.html

### CODE REVIEW

File: tomcat-catalina-ha/7.0.73.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (cnapagoda) [inline tomcat-catalina-ha/7.0.73.wso2v1/pom.xml]: Missing License headers

File: tomcat-servlet-api/7.0.73.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (cnapagoda) [inline tomcat-servlet-api/7.0.73.wso2v1/pom.xml]: Missing License headers

File: tomcat-jsp-api/7.0.73.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (cnapagoda) [inline tomcat-jsp-api/7.0.73.wso2v1/pom.xml]: Missing License headers

File: tomcat-el-api/7.0.73.wso2v1/pom.xml [hunk 4]
```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (cnapagoda) [inline tomcat-el-api/7.0.73.wso2v1/pom.xml]: Missing License headers

File: tomcat-catalina-ha/7.0.73.wso2v1/pom.xml (hunk 1 above)
Comment (NipunaMarcus) [inline tomcat-catalina-ha/7.0.73.wso2v1/pom.xml]: Thank you. Changes Added with the latest commit

File: tomcat-servlet-api/7.0.73.wso2v1/pom.xml (hunk 2 above)
Comment (NipunaMarcus) [inline tomcat-servlet-api/7.0.73.wso2v1/pom.xml]: Thank you. Changes Added with the latest commit

File: tomcat-jsp-api/7.0.73.wso2v1/pom.xml (hunk 3 above)
Comment (NipunaMarcus) [inline tomcat-jsp-api/7.0.73.wso2v1/pom.xml]: Thank you. Changes Added with the latest commit

File: tomcat-el-api/7.0.73.wso2v1/pom.xml (hunk 4 above)
Comment (NipunaMarcus) [inline tomcat-el-api/7.0.73.wso2v1/pom.xml]: Thank you. Changes Added with the latest commit

**Pull request merged**